### ISSUE TITLE
Feature Request - Draw arbitrary connections.

### ISSUE THREAD

**Issue by randomekek** [tensorflow/playground#14]
Looking at the code, the neural net part seems to be general to any arrangement of neurons. I think it's mainly a GUI issue to let the user draw arbitrary neurons and connections? This would be like a 'advanced' UI which you might play with after the current UI, although maybe it might not be that useful for learning purposes.

**Comment by emmanuelrouxfr** [comment]
Dear @randomekek, by reading your issue it seems that you may appreciate the fork we've been working on !

https://coolearning.github.io/playground/

The Coolearning interface allows interactive arbitrary network re-wiring during training, and much more ! Like individual learning rate, activation functions and regularization for each neuron...

See you there :)
